torchmdnet/models/tensornet.py

```diff
@@ -213,6 +229,12 @@ def __init__(
 
         self.reset_parameters()
 
+    def initialize_additional_method(self, method, args):
```

Comment:
This does not have to be a method of the class, you can have it as a free standing function (like "vector_to_skew_tensor")

Reply:
True, fixed!